ddtrace/contrib/openai/patch.py

```diff
@@ -333,7 +335,8 @@ def _patched_make_session(func, args, kwargs):
     This should technically be a ``peer.service`` but this concept doesn't exist yet.
```

Comment:
This still holds true, since we haven't added a peer.service yet right?

Reply:
Yes, for now - I'm double checking with Dusan that we shouldn't be adding peer.service to it

Reply:
double checked, it is still true